{{cookiecutter.project_name}}/src/{{cookiecutter.project_name}}/sample_module.py

```diff
@@ -1,29 +1,29 @@
-"""
-This is a top-level module
-"""
-
-
-class SampleClass:
-    """
-    Sample docstring. Note that init docstrings should be underneath the class and not the init method itself
-    (this looks prettier in sphinx). They still will be rendered correctly in pycharm's quick documentation.
-
-    :param param: some parameter
-    """
-
-    def __init__(self, param: str = None):
-        self.hello = "hello "
-        self.param = param
-
-    def sample_method(self, name: str):
-        """
-        >>> from {{cookiecutter.project_name}}.sample_module import SampleClass
-        >>>
-        >>> greeter = SampleClass()
-        >>> greeter.sample_method("{{cookiecutter.author}}")
-        'hello {{cookiecutter.author}}'
-
-        :param name:
-        :return:
-        """
-        return self.hello + name
+"""
+Top-level module
+"""
+
+
+class SampleClass:
+    """
+    Sample docstring. Note that init docstrings should be underneath the class and not the init method itself
+    (this looks prettier in sphinx). They still will be rendered correctly in pycharm's quick documentation.
+
+    :param param: some parameter
+    """
+
+    def __init__(self, param: str = None):
```

Comment:
need `param: str | None = None`. Generally, pls make sure all poe tasks run through on the generated project. I caught this by running `poe type-check`